### ISSUE TITLE
Feature Request: Multiple buy & sell trigger

### ISSUE THREAD

**Issue by Al0815** [taniman/profit-trailer#112]
Would be very nice to have the option to use multiple trigger to fire an buy/sale if one, some or all of them are true.
Especially if more Indicators like RSI, macd, ... are coming, but also with the existing ones.

Example:
 
```
ALL_buy_strategy1 = LOWBB
ALL_buy_value1 = 5
ALL_trailing_buy1 = 0.35

ALL_buy_strategy2 = EMAGAIN
ALL_buy_value2 = -1


ALL_buy_strategy3 = SMAGAIN
ALL_buy_value3 = -1


# Set A for all must match, [Number|Number2|..] for either of them or set x[number] for X of Y indicators must match 
# All must match (maybe default if not set?)
#ALL_buy_firing = A
# 1 AND 2 must match OR 1 AND 3
#ALL_buy_firing = 1|2+1|3
# 2 of all available must match
#ALL_buy_firing = x2

```

Same goes for sell trigger.

**Comment by taniman** [comment]
I was planning on building this. soon. But let's not make it too complicated. If yo u define multiple triggers. always all must match. At least for version 1

**Comment by taniman** [comment]
Why would you want multiple sell triggers?

**Comment by Al0815** [comment]
Would be reasonable to combine multiple buy and sell triggers. 
So you can have multiple EMA lines, for slow and for fast moving market by example.
Or sort out peaks which affects one trigger, but not the other one. 
There are plenty of possibilities where multiple and/or trigger make sense.

**Comment by taniman** [comment]
Yeah I understand the multiple triggers idea for buying. But for selling I would say. sell as fast as possible. Or are you trying to see if the multi triggers can help with catching a higher price?

**Comment by Al0815** [comment]
Particular for buying I had and/or kombinations in mind, like 2 different sets of ema curves or mix ema with sma, and so on.

But for sell I see the point that it is maybe not worth the effort.

I think multiple buy triggers is more Important and we can get more out of it as with mutliple sell triggers.

**Comment by Jamby93** [comment]
I vote for the idea of combining different triggers with different logic operator, @Al0815 solution with user defined combinations would be perfect but hard to implement, a simple AND/OR on every trigger I guess would be fairly easy to implement and very useful. I would like, for example, to trigger on EMASPREAD OR EMAGAIN, likewise an AND condition would be perfect for something like SMACROSS AND LOWBB

**Comment by taniman** [comment]
There will be no AND/OR it will always be AND

Also this will be implemented under the flag #82
